mayavi/tests/test_mlab_source.py

```diff
@@ -61,6 +61,13 @@ def check_dataset(self):
         self.assertEqual(np.alltrue(vec == v), True)
         self.assertEqual(np.alltrue(sc == s.ravel()), True)
 
+    def test_geometry_filter_works(self):
+        # This tests the bug #1071 and segfaults without the fix on VTK 9.x.
+        x, y, z, v, s, src = self.get_data()
```

Comment:
it looks like none of these variables are used in this test. can we just remove them?